testprojects/src/python/interpreter_selection/python_3_selection_testing/lib_py2.py

```diff
@@ -0,0 +1,18 @@
+# coding=utf-8
+# Copyright 2017 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from __future__ import (absolute_import, division, generators, nested_scopes, print_function,
+                        unicode_literals, with_statement)
+
+
+# A simple example to include in a python 2 binary target
+
+def say_something():
+  print('I am a python 2 library method.')
+  # Return reduce function for testing purposes.
+  # Note that reduce exists as a built-in in Python 2 and
+  # does not exist in Python 3
+  ret = reduce
```

Comment:
Using `assert`s to check for Python version had strange behavior when this is run under pytest so I switched it to reduce for more consistent testing.